src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -153,6 +153,14 @@ public void downloadZippedFile(Environment env, HttpServletRequest request,
 				log.error(errorMessage);
 				downloadStatus.append(errorMessage);
 
+			} catch (Exception e) {
+				String errorMessage = "Migration status for " + site_id + " "
+						+ e.getClass().getName();
+				Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+						.entity(errorMessage).type(MediaType.TEXT_PLAIN)
+						.build();
+				e.printStackTrace();
```

Comment:
Would much rather see log.error(errorMessage,e) - this will log the problem _and_ print the stack trace